mne/report/js_and_css/report.css

```diff
@@ -0,0 +1,19 @@
+#container {
+    position: relative;
+    padding-bottom: 8rem;
+}
+
+#content {
+    margin-top: 90px;
+    scroll-behavior: smooth;
+    position: relative;  /* for scrollspy */
+}
+
+#toc {
+    margin-top: 90px;
+    padding-bottom: 8rem;
+}
+
+footer {
+    margin-top: 8rem;
```

Comment:
just checking: was it intentional to switch from 5 to 8 here? the `.sass` file had `#toc.padding-bottom: 5rem` and `footer.margin-top: 5rem`

Reply:
Yeah 5 was not enough, 8 was determined empirically